src/bubbleSort.js

```diff
@@ -0,0 +1,14 @@
+export default function bubbleSort(array) {
+  if(array.length < 2) return array
+  for (let i = 0; i < array.length; i++) {
```

Comment:
I try to use intention revealing identifiers everywhere - even as loop sentinel values.  Here, I recommend using `inner` and `outer`, or something more meaningful to the search algorithm itself, to increase readability.  One rule of thumb is to select identifiers that make it easier to talk about your code.